### PR TITLE
Some basic data fixes

### PR DESCRIPTION
**Pull request by reubenjh**
Hey @tonywok , just some basic data fixes here. There are a few i'm unsure how you want to approach, i'll comment directly on those

Also i accidentally ran the first commit under my work creds but it seems to have accepted it - let me know if you'd prefer i ammend that

### CODE REVIEW

File: data/en/cards/alpha/cards.yaml [hunk 1]
```diff
@@ -457,6 +457,8 @@
   rules_box: ! |-
     When this Rabble dies on a non-Water site, burrow it instead. They can unburrow as if they had Burrowing.
   type_line: Ordinary Undead scrabble to the surface
+  keywords:
+  - burrowing
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:461]: These don't technically have burrowing but it's consistent for them to have the keyword as far as I can tell as there are a few others who are sort of burrowing-adjacent (if you know what I mean) that do have the keyword

File: data/en/cards/alpha/cards.yaml [hunk 2]
```diff
@@ -544,6 +544,8 @@
   card_type: magic
   artist: dan_seagrave
   name: Bury
+  keywords:
+  - burrowing
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:548]: Same goes for here

File: data/en/cards/alpha/cards.yaml [hunk 3]
```diff
@@ -627,6 +629,8 @@
   card_type: site
   artist: adam_burke
   name: City of Agartha
+  keywords:
+  - burrowing
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:633]: ☝️

File: data/en/cards/alpha/cards.yaml [hunk 4]
```diff
@@ -1077,6 +1083,8 @@
     Choose a square block of four locations of which at least one is an Earth site.
 
     Burrow all ground minions on affected sites without Water. You may rearrange the placement of those sites within the block.
+  keywords:
+  - burrowing
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:1087]: ☝️

File: data/en/cards/alpha/cards.yaml [hunk 5]
```diff
@@ -1982,17 +1996,6 @@
   type_line: Wind whirls and Water swirls in one Elite Site.
   water_threshold: '1'
   wind_threshold: '1'
-- identifier: lightning_bolt
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:1985]: There was an entry for lightening bolt and thunderbolt, i don't actually know which one they are going with but figured it was probably the one with a different name to mtg's lightning bolt 😆

File: data/en/cards/alpha/cards.yaml [hunk 6]
```diff
@@ -2224,6 +2229,8 @@
     This Army must be summoned to any of the two middle sites.
 
     It can carry out ranged attacks on enemies on any site in the Realm.
+  keywords:
+  - range_x
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:2233]: Doesn't technically have the range keyword but performs ranged attacks

File: data/en/cards/alpha/cards.yaml [hunk 7]
```diff
@@ -3412,6 +3423,8 @@
     Tap, tap an allied minion here -> This Ballista deals 3 ranged combat damage to an Avatar or minion up to two steps away in a cardinal direction.
 
     This Ballista can't be picked up, but an Avatar or minion may carry it as it moves.
+  keywords:
+  - range_x
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:3427]: Same here ☝️

File: data/en/cards/alpha/cards.yaml [hunk 8]
```diff
@@ -3631,21 +3644,6 @@
   type_line: Ordinary Magic of radial violence
   mana_cost: '3'
   fire_threshold: '1'
-- identifier: spire
```
Comment (reubenjh) [inline data/en/cards/alpha/cards.yaml:3634]: Typo / duplicate from spire lich

File: data/en/cards/alpha/cards.yaml [hunk 9]
```diff
@@ -290,14 +290,14 @@
...
-  earth_threshold: '1'
+  fire_threshold: '1'
 - identifier: ball_lightning
   rarity: elite
   card_type: magic
   artist: alan_pollack
   name: Ball Lightning
   rules_box: ! |-
-    Your Spellcaster lanches this Ball in a cardinal direction. It deals 6 damage to the first Avatar or minion it hits. Excess damage carries on to whatever it hits next here or on following site.
+    Your Spellcaster launches a Fireball that travels in a cardinal direction. It deals 6 damage to the first Avatar or minion it hits. Excess damage carries on to whatever it hits next here or on following site.
```
Comment (tonywok) [inline data/en/cards/alpha/cards.yaml:300]: I think this should read as follows based on the [latest beta file](http://cloud-3.steamusercontent.com/ugc/1853799038721785053/64FF415C68D8229006DC605FC13B5FA8BCFA9B30/) which is soon to be replaced.

> Your Spellcaster launches this Ball in a cardinal direction. It deals 6 damage to the first Avatar or minion it hits. Excess damage carries on to whatever it hits next here or on following sites.

But it does appear I've spelled launches incorrectly.

Comment (reubenjh) [reply data/en/cards/alpha/cards.yaml]: Fixed 👍

**Review by tonywok — APPROVED** [review]
Thanks 🚀

**Pull request merged**